infra/main.bicep

```diff
@@ -32,16 +44,28 @@ var resourceToken = toLower(uniqueString(subscription().id, environmentName, loc
 var tags = { 'azd-env-name': environmentName }
 
 // Organize resources in a resource group
-resource rg 'Microsoft.Resources/resourceGroups@2021-04-01' = {
+resource resourceGroup 'Microsoft.Resources/resourceGroups@2021-04-01' = {
   name: !empty(resourceGroupName) ? resourceGroupName : '${abbrs.resourcesResourceGroups}${environmentName}'
   location: location
   tags: tags
 }
 
+resource openAiResourceGroup 'Microsoft.Resources/resourceGroups@2021-04-01' existing = if (!empty(openAiResourceGroupName)) {
```

Comment:
Just reading the code, are we moving to a model where there is a RG per resource type or will they all just be mapped to the AZURE_RESOURCE_GROUP by default? I assume this explicit mapping is needed for the scenario where users are bringing their own resources that might not be in the same group?

Reply:
We default to a single resource group, but allow the user to override with an existing resource group.  

The scenario we don't support is creating the solution from scratch using more than one resource group.  The "other" resource groups need to already exist.